extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -876,6 +876,13 @@ export class AmpAutocomplete extends AMP.BaseElement {
         this.resetActiveElement_();
         newActiveElement.classList.add('i-amphtml-autocomplete-item-active');
         newActiveElement.setAttribute('aria-selected', 'true');
+        if (!newActiveElement.hasAttribute('id')) {
+          newActiveElement.setAttribute('id', 'autocomplete-selected');
```

Comment:
If there is already an element on the page with the `autocomplete-selected` id on the page, this will collide with it. Let's generate an id similar to how amp-accordion does.

https://github.com/ampproject/amphtml/blob/0e180322a141e9488226b80f600ded1f53823151/extensions/amp-accordion/0.1/amp-accordion.js#L106-L114

Reply:
Ah, this is a much better solution. The random number doesn't **ensure** uniqueness, though, since there is a very small chance the same number could be generated if there were very many autocompletes on the page, right?

Reply:
This is true. For now this is good enough, and maybe in a future PR we can create a method in `src/dom.js` to generate a truly unique id.